Mathlib/Logic/Equiv/PartialEquiv.lean

```diff
@@ -560,6 +560,19 @@ theorem ofSet_coe (s : Set α) : (PartialEquiv.ofSet s : α → α) = id :=
 theorem ofSet_symm (s : Set α) : (PartialEquiv.ofSet s).symm = PartialEquiv.ofSet s :=
   rfl
 
+/-- `Function.const` as a `PartialEquiv`.
+  It consists of two constant maps in opposite directions. -/
+@[simps]
+def single (a : α) (b : β) : PartialEquiv α β where
+  toFun := Function.const α b
+  invFun := Function.const β a
+  source := {a}
+  target := {b}
+  map_source' := fun _ _ ↦ by rfl
+  map_target' := fun _ _ ↦ by rfl
+  left_inv' := fun a' ha'  ↦ by rw [eq_of_mem_singleton ha']; rfl
```

Comment:
Is there some explicit lemma that `rfl` is calling? Can you try if `rw?` suggests something?